monitoring/task-definitions/monitoring.json

```diff
@@ -0,0 +1,132 @@
+[
+      {
+        "name":"s3_download",
```

Comment:
can you ensure the execution order of the containers?

Reply:
so how will we make sure this works properly all the time?

Reply:
So for the first time it's not a problem. For the updates I think we can deploy them with concourse in order to make sure it's using the latest config

Reply:
why do you say it won't be a problem the first time? Afaik if the Prometheus container starts before the `s3_download` one, it won't have any configuration to use and it will either fail or malfunction.

Reply:
maybe you can trigger a reload to prometheus and grafana after the s3 command completes. Both Prometheus and Alertmanager can be told to reload the config either via a SIGHUP signal or an http call to `/-/reload` (seen here https://www.robustperception.io/reloading-prometheus-configuration). And cloudwatch-exporter supports it too: https://github.com/prometheus/cloudwatch_exporter#reloading-configuration

This would ensure that those processes are always running the latest configuration.

Reply:
ok great

Reply:
Fixed in a different way. I've used volumesFrom instead and after some tests it seems working as expected